proms-resources/frontend/src/main/frontend/src/proms/WelcomeMessageConfiguration.jsx

```diff
@@ -0,0 +1,177 @@
+  },
+  saveButton: {
+    marginTop: theme.spacing(3),
+  },
+}));
+
+function WelcomeMessageConfiguration() {
+  const classes = useStyles();
+
+  const [ path, setPath ] = useState();
+  const [ welcomeMessage, setWelcomeMessage ] = useState();
+
+  // Status tracking values of fetching/posting the data from/to the server
+  const [ error, setError ] = useState();
+  const [ isSaved, setIsSaved ] = useState(false);
+  const [ fetched, setFetched ] = useState(false);
+
+  const appName = document.querySelector('meta[name="title"]')?.content;
+
+  const theme = useTheme();
+  const breakpoint = useMediaQuery(theme.breakpoints.down("sm"));
+
+  // Fetch saved admin config settings
+  let getWelcomeMessage = () => {
+    fetch('/Proms/WelcomeMessage.json')
+      .then((response) => response.ok ? response.json() : Promise.reject(response))
+      .then((json) => {
+        setFetched(true);
+        setPath(json["@path"]);
+        setWelcomeMessage(json.text || "");
+      })
+      .catch(setError);
+  }
+
+  // Submit function
+  let handleSubmit = (event) => {
+
+    // This stops the normal browser form submission
+    event && event.preventDefault();
+
+    // Build formData object.
+    // We need to do this because sling does not accept JSON, need url encoded data
+    let formData = new URLSearchParams();
+    formData.append('text', welcomeMessage);
+
+    // Use native fetch, sort like the XMLHttpRequest so no need for other libraries.
+    fetch("/Proms/WelcomeMessage",
+      {
+        method: 'POST',
+        headers: {
+          'Accept': 'application/json',
+          'Content-Type': 'application/x-www-form-urlencoded'
+        },
+        body: formData
+      })
+      .then((response) => {
+
+        // Important note about native fetch, it does not reject failed
+        // HTTP codes, it'll only fail when network error
+        // Therefore, you must handle the error code yourself.
+        if (!response.ok) {
+          throw Error(response.statusText);
+        }
+
+        setIsSaved(true);
+      })
+      .catch(setError);
+  }
+
+  if (!fetched) {
+    getWelcomeMessage();
+  }
+
+  return (
+    <Card>
+      <CardHeader
+        title="Welcome Message text"
+        titleTypographyProps={{variant: "h6"}}
+      />
+      <CardContent>
+        {error && <Typography color='error'>{error}</Typography>}
+        <Typography>
+          In the welcome message, use APP_NAME to refer to the name configured for the application.
+          On the Patient identification screen, all occurrences of APP_NAME will appear as {appName}.
+        </Typography>
+        <form onSubmit={handleSubmit}>
+          <List>
+            <ListItem key="form">
+              { welcomeMessage != undefined &&
+                <Grid
+                  container
+                  spacing={2}
+                  direction={breakpoint ? "column" : "row"}
+                  justifyContent="center"
+                  alignItems="flex-start"
```

Comment:
We should try to make the two boxes the same height for a more balanced look.
```suggestion
                  alignItems="stretch"
```

Reply:
Height match implemented in 470b0a2.